iml-agent/src/action_plugins/action_plugin.rs

```diff
@@ -41,7 +41,8 @@ pub fn create_registry() -> action_plugins::Actions {
         .add_plugin(
             "configure_ntp",
             action_configure::update_and_write_new_config,
-        );
+        )
+        .add_plugin("create_ltuer_conf", ltuer::create_ltuer_conf);
```

Comment:
While you are here, could you sort them all?

Reply:
Maybe in separate PR